app/models/cda/base_model.rb

```diff
@@ -0,0 +1,12 @@
+module Cda
+  class BaseModel < ActiveResource::Base
+    self.site = CourtDataAdaptor::Resource::V2.api_url
```

Comment:
I've used the same value for `self.site`, but I'd like to change it.
I think we should not use the the `CourtDataAdaptor::Resource::V2.api_url` : `Cda::BaseModel` help to create resources,  and we are using the site url of a resource that we are deprecating.

I think we should not reuse code in `CourtDataAdaptor::Resource`